server/lib/payments.js

```diff
@@ -832,6 +833,10 @@ export const getHostFee = async (order, host = null) => {
 };
 
 export const isPlatformTipEligible = async (order, host = null) => {
+  if (order.collective?.data?.disablePlatformTips) {
```

Comment:
What is this?

Reply:
This is the edge-case I added for BRL contributions related to Stripe's `aplication_fee` limitation.

Reply:
Can we remove this now that we're landing https://github.com/opencollective/opencollective-api/pull/7020?